template_to_pdf/__init__.py

```diff
@@ -2,15 +2,15 @@
 import os.path
 import sys
 
-from jinja2 import Environment, FileSystemLoader, select_autoescape
+from jinja2 import Environment, ChoiceLoader, FileSystemLoader, select_autoescape
 from weasyprint import HTML
 
 
 class PdfRenderer:
     template_filename = None
 
     def __init__(self, templates_path=None):
-        default_templates_path = glob.glob('{}/*/templates/'.format(sys.path[0]))
+        default_templates_path = (glob.glob('{}/*/templates/'.format(path)) for path in sys.path[:2])
```

Comment:
Could not you use a relative to application paths instead of 'PYTHONPATH's?

Reply:
No, because this lib doesn't know what is the actual application path. Any project could install this lib, create a template folder inside it and use this class. 

I understand searching for the sys.path might lead to bugs, but I think we can expect these bugs happens and then figure out a better way to do this. Because as the code is now it's working as expected.